bats.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: bats
+  version: 1.11.0
+  epoch: 0
+  description: Bash Automated Testing System
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - bash
+      - coreutils
+      - ncurses
+      - parallel
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/bats-core/bats-core
+      tag: v${{package.version}}
+      expected-commit: 5da66876b8b619235aee1eb3e54954eaca88059b
+
+  - runs: |
+      ./install.sh  "${{targets.contextdir}}/usr"
+
+subpackages:
+  - name: bats-doc
+    pipeline:
+      - uses: split/manpages
+    description: bats manpages
+
+update:
+  enabled: true
+  github:
+    identifier: bats-core/bats-core
+    strip-prefix: v
+    use-tag: true
```

Comment:
Looks like BATS uses releases, should we prefer those over the tags?

```suggestion
```

Reply:
@stormqueen1990 already pushed that!